### ISSUE TITLE
Can't use in Next.js 13 appDir

### ISSUE THREAD

**Issue by alvarlagerlof** [sanity-io/code-input#35]
Fails with 

```
error - ./node_modules/@sanity/code-input/lib/index.esm.js:1:1244
Module not found: Can't resolve 'styled-components'
```

And if I install "styled-components":

```
Maybe one of these should be marked as a client entry with "use client":
  node_modules/@sanity/code-input/lib/index.esm.js
```

**Comment by alvarlagerlof** [comment]
Update, got it to load, but all of them look empty.

![image](https://user-images.githubusercontent.com/14835120/205396754-303aed43-c2cc-4cd9-8a46-d656a264db44.png)

**Comment by alvarlagerlof** [comment]
Looks to be fixed in 3.0.1.
